src/pages/Integrations/Spotify/SpotifyScreen/SpotifyScreen.tsx

```diff
@@ -21,6 +21,7 @@ import SpotifyWidgetPro from '../../../../components/Integrations/Spotify/Widget
 import SpAudioFeatures from '../../../../components/Integrations/Spotify/Widgets/SpotifyWidgetPro/SpAudioFeatures/SpAudioFeatures';
 import SpotifyTriggerTable from '../../../../components/Integrations/Spotify/Widgets/SpotifyWidgetPro/SpTriggerTable';
 import SpPlaylist from '../../../../components/Integrations/Spotify/Widgets/SpotifyWidgetPro/SpPlaylist';
+import Layout from '../../../../components/AudioAnalysis/Layout';
```

Comment:
maybe we can have a more descriptive name than layout :)

Reply:
Done